archiver.go

```diff
@@ -400,12 +391,17 @@ func (a *Archiver) pushChangesToRootedRepositories(
 		cancel()
 	}
 
-	if len(changes) == 0 {
+	logger.Debugf("update repository references started")
```

Comment:
I believe this logger line and the next one belong to the go routine 
```go
logger.Debugf("update repository references started")
r.References = updateRepositoryReferences(r.References, cs, ic)
for _, ref := range r.References {
        ref.Repository = r
}

logger.Debugf("update repository references finished")
```

Reply:
Now updating references is split in two. One in memory that is done in the goroutine and the other part is updating the database. I'm more interested in checking when it updates the database (how many times and the time it takes) as it's what was causing problems.